packages/language-server/test/snapshots/xml-view-diagnostics/utils.ts

```diff
@@ -0,0 +1,106 @@
+import { resolve, sep } from "path";
+import { Diagnostic } from "vscode-languageserver-types";
+import { TextDocument } from "vscode-languageserver";
+import { readJsonSync, readFileSync } from "fs-extra";
+import { expect } from "chai";
+import { sortBy, forEachRight } from "lodash";
+import { generateModel } from "@ui5-language-assistant/test-utils";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getXMLViewDiagnostics } from "../../../src/xml-view-diagnostics";
+
+export const INPUT_FILE_NAME = "input.xml";
+export const OUTPUT_LSP_RESPONSE_FILE_NAME = "output-lsp-response.json";
+export const OUTPUT_RANGES_FILE_NAME = "output-ranges.xml";
```

Comment:
I though we could have to ranges inside the `input.xml`. It's not part of the input but it would prevent copying the input file, specifying the ranges in the response json file (instead only specifying the range number) and syncing it between the 2 output files.

Reply:
I think I still want to print the whole LSP response,but combining the expected range markers in the input is a good idea, I will try it.